content/SCALE/SCALECLIReference/Account/CLIuser.md

```diff
@@ -534,30 +491,26 @@ It is recommended to use "admin." Root user access is a deprecated method.
 
 ### Shell_Choices Command
 
-The `shell_choices` command returns the shell choices available to user accounts.
+The `shell_choices` command returns the shell choices available to user accounts. 
 
 {{< expand "Using the Shell_Choices Command" "v" >}}
-
 #### Description
-
-Enter the command with no additional properties to return shell choices available to all users.
-
-Additional shell choices are available to users with administrative privileges (members of the **builtin_administrators** group).
```

Comment:
I think we do need to offer both versions of the command in some form. If the user has administrative privledges they can access the additional options, but if not they cannot. It isn't intuitive to find those additional options or to know they are there unless we explain it. 

```
[qe-scale-04]> account user shell_choices group_ids=["builtin_administrators"]
+----------------------+-----------------+
|    /usr/sbin/nologin | nologin         |
|         /usr/bin/cli | TrueNAS CLI     |
| /usr/bin/cli_console | TrueNAS Console |
|        /usr/bin/bash | bash            |
|       /usr/bin/rbash | rbash           |
|        /usr/bin/dash | dash            |
|          /usr/bin/sh | sh              |
|        /usr/bin/tmux | tmux            |
|         /usr/bin/zsh | zsh             |
+----------------------+-----------------+
```
```
[qe-scale-04]> account user shell_choices
+-------------------+---------+
| /usr/sbin/nologin | nologin |
|     /usr/bin/bash | bash    |
|    /usr/bin/rbash | rbash   |
|     /usr/bin/dash | dash    |
|       /usr/bin/sh | sh      |
|     /usr/bin/tmux | tmux    |
|      /usr/bin/zsh | zsh     |
+-------------------+---------+
```

Reply:
Short answer is no. We can revisit how we want to do specific examples, like a command string that configures HA networking or creating a VLAN or LAGG interface and then include the command examples for this, but it is not necessary in this case. One example command output is sufficient.

Reply:
Maybe not a command example, but at least a sentence in the description